tests/v2/validation/provisioning/rke1/provisioning_node_driver.go

```diff
@@ -64,5 +66,13 @@ func TestProvisioningRKE1Cluster(t *testing.T, client *rancher.Client, provider
 	assert.NotEmpty(t, podResults)
 	assert.Empty(t, podErrors)
 
+	if len(psact) > 0 && (psact == string(provisioning.RancherPrivileged) || psact == string(provisioning.RancherRestricted)) {
```

Comment:
So Markus, we don't need the first check cause after `&&` we're accepting it's not an empty string either way. 

So only this would be enough for this if check:
`psact == string(provisioning.RancherPrivileged) || psact == string(provisioning.RancherRestricted)`

Reply:
Good point, the first check really is not necessary.

Reply:
Resolved.